CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,29 @@
+This code of conduct outlines our expectations for all those who want to participate or interact with the Kord project via GitHub Issues, Discussions, Pull Requests, our Discord server, or similar
+### How to behave
+The following behaviors are expected and requested of all community members:
+
+- Participate in an authentic and active way. In doing so, you contribute to the health and longevity of this community.
+- Exercise consideration, respect, and empathy in your speech and actions. Remember, we have all been through different stages of learning when adopting technologies.
+- Refrain from demeaning, discriminatory, or harassing behavior and speech.
+- Disagreements on things are fine, argumentative behavior or trolling are not.
+
+### How not to behave
+- Do not perform threats of violence or use violent language directed against another person.
+- Do not make jokes of sexist, racist, homophobic, transphobic, ableist, or otherwise discriminatory nature, or use language of this nature.
+- Do not post or display sexually explicit or violent material.
+- Do not post or threaten to post other people’s personally identifying information ("doxxing").
+- Do not make personal insults, particularly those related to gender, sexual orientation, race, religion, or disability.
+- Do not engage in sexual attention. This includes sexualized comments or jokes and sexual advances.
+- Do not advocate for, or encourage, any of the above behavior.
+
+Please take into account that online communities bring together people from many different cultures and backgrounds. It's important to understand that sometimes the combination of cultural differences and online interaction can lead to misunderstandings. That is why having empathy is crucial.
+
+### How to report issues
+If you want to report another member's violations or violations of this Code of Conduct and/or need assistance because of that person's behavior, feel free to contact one of the Admins (schlaubi, lustiger.lurch) on [our Discord Server](https://discord.gg/6jcx5ev)
+
+### Consequences of Unacceptable Behavior
+Unacceptable behavior from any community member will not be tolerated. Anyone asked to stop unacceptable behavior is expected to comply immediately. If a community member engages in unacceptable behavior, the Kord maintainers may take any action they deem appropriate, up to and including a temporary or permanent expulsion from the community without warning.
+
+### License and attribution
+The license is based on [The Citizen Code of Conduct](https://github.com/stumpsyn/policies/blob/master/citizen_code_of_conduct.md) is distributed by Stumptown Syndicate under a Creative Commons Attribution-ShareAlike license.
+We also took inspiration from the [JetBrains](https://github.com/jetbrains#code-of-conduct) Code of Conduct

```

Comment:
It seems you didn't update this link - it still looks like you're linking to the JetBrains site, rather than their Code of Conduct.

Reply:
This is the Link to their code of conduct
![image](https://github.com/user-attachments/assets/54e4a33c-acfc-4b60-a26d-76d73ee3aeaa)

Reply:
yes, I know where the link goes literally, you've just been like

the [JetBrains](https://github.com/jetbrains#code-of-conduct) Code of Conduct

instead of

the [JetBrains Code of Conduct](https://github.com/jetbrains#code-of-conduct)

Reply:
oh that's what you meant, gonna fix that thx